### PR TITLE
Added iOS 8 Locksaver for OSX

### PR DESCRIPTION
**Pull request by specious**
Successor to cask `ios7-screensaver`.

### CODE REVIEW

File: Casks/ios8-screensaver.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+cask :v1 => 'ios8-screensaver' do
```
Comment (vitorgalvao) [inline Casks/ios8-screensaver.rb]: Change this to `cask :v1 => 'ios-saver' do`, to be better in line with [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_token_reference.md).

The file’s name also needs to be changed to `ios-saver.rb`.

File: Casks/ios8-screensaver.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  version :latest
+  sha256 :no_check
+
+  # site90.net is the official download host per the vendor homepage
+  url 'http://dlstatsios7.site90.net/download.php',
+    :referer => 'http://littleendiangamestudios.com/project/ios-8-screen-saver/'
+  name 'iOS 8 Lockscreen for OSX'
+  homepage 'http://littleendiangamestudios.com/project/ios-8-screen-saver/'
+  license :unknown
```
Comment (vitorgalvao) [inline Casks/ios8-screensaver.rb]: `:gratis`.

**Comment by specious** [comment]
Thank you for reviewing this package.  I've made the adjustments.

**Comment by vitorgalvao** [comment]
Thank you.

**Pull request merged**